src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -199,12 +199,24 @@ button.NormalAnnouncementBubble--deleteMenuItem {
 }
 
 .NormalAnnouncementBubble--readReceipts--tooltip {
+  cursor: pointer;
   .text--semi-bold();
   padding-right: @size_2xs;
   padding-left: @size_2xs;
   white-space: pre-wrap;
 }
 
+Button.NormalAnnouncementBubble--readReceipts--button {
```

Comment:
I think we probably want this to be `.Button` with a dot, though I'm also wondering if Ashley's recent PR has this covered already!

https://github.com/Clever/components/pull/725

https://github.com/Clever/components/blob/3a8143c4c2d592de73e8dc7799479a881736de7a/src/Button/Button.less#L144-L151

Reply:
Nice ill drop it!

Reply:
Actually it isnt working without this code, and i think its because ashley added that to the link/linkPlain buttons and not the other types and Im using a plain here. Is the intention that the focus behavior should be the same on all button types?

Reply:
Also TIL that casing isn't enforced for element selectors. So that's why `Button` worked in this case without the `.`. Even `BUtton` works. But i know we discussed using `.Button` from now on

Reply:
Ooh great catch -- you're right that Ashley's update was for link/linkPlain buttons and not other types. On that note, applying a one-off fix here seems reasonable!

I do think it's worth checking in with Nick after the fact whether we also want to apply a global fix to the other `Button` types